contracts/src/v0.8/tests/StreamsLookupUpkeep.sol

```diff
@@ -93,7 +93,7 @@ contract StreamsLookupUpkeep is AutomationCompatibleInterface, StreamsLookupComp
       return (false, data);
     }
     uint256 timeParam;
-    if (keccak256(abi.encodePacked(timeParamKey)) == keccak256(abi.encodePacked("feedIdHex"))) {
+    if (keccak256(abi.encodePacked(feedParamKey)) == keccak256(abi.encodePacked("feedIdHex"))) {
```

Comment:
Hey @FelixFan1992 , i know this is a simple change, and this is all about this PR, I wish I could just approve. Can you help me understand what are timeParamKey and feedParamKey, why this change? Thanks~

Reply:
sure. timeParamKey is either `blockNumber` or `timestamp`
feedParamKey is either `feedIdHex` or `feedIDs`. these are used to differentiate v2 and v3